app/views/books/show.html.haml

```diff
@@ -55,7 +55,10 @@
           %strong= @book.name
         %p
           Статус:
-          %strong= @book.status
+          - if @book.status == 'in'
+            %img{src: "/uploads/green2.png", class: 'status'}
+          - else
+            %img{src: "/uploads/red.png", class: 'status'}
```

Comment:
%img{src: @book.status == 'in' ? "/uploads/green2.png" : "/uploads/red.png", class: 'status'}